aspnetcore/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/Views/Shared/_Layout.cshtml

```diff
@@ -29,7 +29,7 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a asp-area="" asp-controller="Movies" asp-action="Index" class="navbar-brand">MvcMovie</a>
+                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">MvcMovie</a>
```

Comment:
I changed this because it no Movies controller or views exist and when you click the nav bar, you get an exception. Should I change it back?